packages/components/src/utils/reduce-motion.js

```diff
@@ -3,6 +3,10 @@
  *
  * @param {'transition' | 'animation' | string} [prop='transition'] CSS Property name
  * @return {string} Generated CSS code for the reduced style
+ *
+ * @deprecated Write your own media query instead,
```

Comment:
Not in this PR, but should we also refactor the existing usages? The best way to ensure that there will be new usage of this deprecated utility is to keep using it in the Gutenberg codebase.

Reply:
+1 ^

Reply:
It's just a matter of prioritization. Ideally they should be rewritten, but this one is pretty benign compared to the million other things we need to rewrite 😭 Though, it's probably a good first issue for new contributors so I'll post an issue.

Reply:
No urgency here, an issue sounds good 👍